source/support/connect-to-govwifi-using-an-iphone-or-ipad.html.erb

```diff
@@ -24,21 +24,15 @@ description: Follow these instructions to connect your iPhone or iPad to GovWifi
           <li><%= partial 'shared/copy/password_instructions' %></li>
           <li>
             <p class="govuk-body">You'll be asked to confirm that the network presents the correct certificate. This protects you against fake networks which may harm your privacy.</p>
-            <p class="govuk-body">The correct certificate is shown below. It's called <strong>wifi.service.gov.uk</strong> and is issued by <strong>GeoTrust DV SSL SHA256 CA</strong>.</p>
-            <%= image_tag "troubleshooting/ios-step-3.jpg", class: "img-fluid", alt: "Screenshot of WiFi certificate panel on iOS" %>
+            <p class="govuk-body">The correct certificate is called <strong>wifi.service.gov.uk</strong> and is issued by <strong>GeoTrust RSA CA 2018</strong>.</p>
           </li>
           <li>
             <p class="govuk-body">Select <strong>More Details</strong> and scroll down.</p>
-            <%= image_tag "troubleshooting/ios-step-4.jpg", class: "img-fluid", alt: "Screenshot of WiFi certificate details panel on iOS" %>
-          </li>
-
-          <li>
-            <p class="govuk-body govuk-!-margin-top-7">Check that the SHA-1 Fingerprint matches <strong>76 29 76 AA 3D CE A8 5C D8 3D 72 0A B8 10 32 93 74 A5 15 5A</strong>.
+            <p class="govuk-body govuk-!-margin-top-7">Check that the SHA-1 Fingerprint matches <strong>37 CE B5 7A 3F 38 EB 7C 66 A2 4E 90 BB 11 71 D7 BC 4D 65 1A</strong>.
```

Comment:
you could make this a new list item to avoid the margin override

Reply:
Fixed by using the same pattern that is used on [/source/support/connect-to-govwifi-using-a-windows-device.html.erb](/source/support/connect-to-govwifi-using-a-windows-device.html.erb)